Feature/ufoeval adc

ADCollard opened the pull request:
Fix eva YAMLs to be consistent with changes in [this issue](https://github.com/JCSDA-internal/eva/pull/80).  Bug fix to `ush/ufoeval/run_ufo_hofx_test.sh`

ADCollard committed:
Changes to yaml to reflect eva changes

ADCollard committed:
Bug fixes

CoryMartin-NOAA commented on ush/eva/jedi_gsi_compare_conv.yaml:
```diff
@@ -279,18 +279,20 @@ diagnostics:
             - mean
             - max
             - std
             - name
```
this one still has `name` does that work?

ADCollard replied:
> this one still has `name` does that work?

@CoryMartin-NOAA Full disclosure, I was not able to test this YAML as none of the conventional YAMLs are giving valid output file right now.  So this is a bug.

I'm changing this to a draft PR until I find a way to test this one.

But yes, `name` needs to be removed.

nicholasesposito commented on ush/eva/jedi_gsi_compare_conv.yaml:
```diff
@@ -368,18 +371,19 @@ diagnostics:
-              @CHANNELKEY@
+            fields:
+              - field_name: experiment::HofXDiff::${variable}
+                @CHANNELKEY@
```
Looking over the previous commit that @CoryMartin-NOAA showed me yesterday, it looks like this should be 
`                channel: ${channel}`
but I might be wrong. What is the preferred way to do this?

CoryMartin-NOAA replied:
This is because the python script decides to put that in if it needs to

CoryMartin-NOAA committed:
Merge branch 'develop' into feature/ufoeval_ADC

CoryMartin-NOAA committed:
Merge branch 'develop' into feature/ufoeval_ADC

ADCollard committed:
Fix and tidy up eva yaml generation, update test input directory

ADCollard committed:
Merge branch 'feature/ufoeval_ADC' of https://github.com/NOAA-EMC/GDASApp into feature/ufoeval_ADC

ADCollard commented:
I have fixed the issues with issues with the conventional eva yaml and tidied up directory structure.  Also updated the date for testing in `ush/ufoeval/run_ufo_hofx_test.sh`.

Ready for review.
